build/misc.tcl

```diff
@@ -895,6 +895,10 @@ respond "*" ":midas /t device;jobdev glp_sysen2;xgpdev\r"
 respond "with ^C" "GLP==1\r\003"
 expect ":KILL"
 
-# KL10 frontend directory
```

Comment:
It'd be nicer to fix this in the previous commit (and maybe say "Tool" at the end as you've done in programs.md).

Reply:
Thanks, I must have added this to the wrong commit.